components/global-header/help.jsx

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// # Global Header Help Component
+// Implements the [Global Header Help design pattern](https://www.lightningdesignsystem.com/components/global-header/#Help) in React.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import assign from 'lodash.assign';
+import Button from '../button';
+import Popover from '../popover';
+
+import { GLOBAL_HEADER_HELP } from '../../utilities/constants';
+
+const propTypes = {
+	/**
+	 * **Assistive text for accessibility**
+	 * * `triggerButton`: Assistive text for the GlobalHeaderHelp trigger button. The default is `Help and Training`.
+	 */
+	assistiveText: PropTypes.shape({
+		triggerButton: PropTypes.string,
+	}),
+	/**
+	 * A `Popover` component. The props from this popover will be merged and override any default props.
+	 */
+	popover: PropTypes.node,
+};
+
+/**
+ * A GlobalHeaderHelp component.
+ */
+class GlobalHeaderHelp extends React.Component {
+	render() {
+		const buttonAriaProps = {
+			'aria-haspopup': true,
+		};
+		const popoverProps = assign(
+			{
+				align: 'bottom',
+				body: <span />,
+				heading: 'Help and Training',
```

Comment:
Same as above. Please move to Popover body with linked id, so that `undefined` doesn't need to be passed in to override.